vcpkg-python-scripts/vcpkg_python_functions.cmake

```diff
@@ -94,4 +94,22 @@ function(vcpkg_python_build_and_install_wheel)
 
   vcpkg_python_build_wheel(${opts} SOURCE_PATH "${arg_SOURCE_PATH}" OUTPUT_WHEEL WHEEL OPTIONS ${arg_OPTIONS})
   vcpkg_python_install_wheel(WHEEL "${WHEEL}")
-endfunction()
\ No newline at end of file
+endfunction()
+
+function(vcpkg_python_test_import)
+    cmake_parse_arguments(
+        PARSE_ARGV 0
+        "arg"
+        ""
+        "MODULE"
+        ""
+    )
+
+  message(STATUS "Testing package!")
+  set(ENV{PYTHONPATH} "${CURRENT_PACKAGES_DIR}/lib/python${PYTHON3_VERSION_MAJOR}.${PYTHON3_VERSION_MINOR}/site-packages")
+  vcpkg_execute_required_process(COMMAND "${PYTHON3}" -c "from ${arg_MODULE} import *"
+    LOGNAME "python-test-import-${TARGET_TRIPLET}"
+    WORKING_DIRECTORY "${CURRENT_BUILDTREES_DIR}"
+  )
```

Comment:
I think you need to add a call for `os.add_dll_directory` on windows. Otherwise dlls in `${CURRENT_PACKAGES_DIR}/bin` are not visible. Unix is fine if rpath is getting adjusted by the rpath fixup.

Reply:
I don't have a windows to test here, so I'll leave that to fixup for whoever has got the means to test

Reply:
I think it's easier to continue with this under the new organisation, I am having some troubles to build here at the moment, and I'd prefer to invest time into research under the new structure